tests_basic/test_winstonlutz.py

```diff
@@ -48,6 +48,9 @@ def test_bb_shift_instructions(self):
         move = self.wl.bb_shift_instructions(couch_vrt=-2, couch_lat=1, couch_lng=100)
         self.assertTrue("RIGHT" in move)
         self.assertTrue("VRT" in move)
+    
+    def test_json(self):
```

Comment:
I appreciate writing a test, but this test is nearly un-failable. Testing the output type and at least one value would be helpful. E.g. `self.AssertEqual(wl.results_data["Gantry iso size"], wl.gantry_iso_size)`

Reply:
Agreed